src/api/repository/user_repository.py

```diff
@@ -106,3 +106,57 @@ def _insert_user(self, cur, user: User, address_id: int) -> dict:
         )
         cur.execute(user_query, user_values)
         return cur.fetchone()
+
+    def get_user_by_id(self, user_id: int) -> Optional[User]:
```

Comment:
same here,

get_user_by_id(id: int) -> get_user(id: int)

Reply:
Changed in all places